makefile

```diff
@@ -67,17 +68,28 @@ start_sentinel: ${BINARY}
 		conf=${TMP}/sentinel$$port.conf;\
 		touch $$conf;\
 		echo '' >  $$conf;\
-		echo 'sentinel monitor                 master1 127.0.0.1 ${PORT} 2' >> $$conf;\
-		echo 'sentinel down-after-milliseconds master1 5000'                >> $$conf;\
-		echo 'sentinel failover-timeout        master1 30000'               >> $$conf;\
-		echo 'sentinel parallel-syncs          master1 1'                   >> $$conf;\
+		echo 'sentinel monitor                 ${HA_GROUP_NAME} 127.0.0.1 ${PORT} 2' >> $$conf;\
+		echo 'sentinel down-after-milliseconds ${HA_GROUP_NAME} 5000'                >> $$conf;\
+		echo 'sentinel failover-timeout        ${HA_GROUP_NAME} 30000'               >> $$conf;\
+		echo 'sentinel parallel-syncs          ${HA_GROUP_NAME} 1'                   >> $$conf;\
 		${BINARY} $$conf\
 			--daemonize yes\
 			--pidfile   ${TMP}/redis$$port.pid\
 			--port      $$port\
 			--sentinel;\
 	done
 
+wait_for_sentinel:
+	@for port in ${SENTINEL_PORTS}; do\
+		while : ; do\
+			if [ $$(${REDIS_CLIENT} -p $${port} SENTINEL SLAVES ${HA_GROUP_NAME} | wc -l) -gt 1 ]; then\
+				break;\
+			fi;\
+			echo 'waiting for ready to sentinel tests...';\
```

Comment:
`Waiting for Redis sentinel to be ready` would be more correct.

Reply:
I'm sorry for my bad English. I fixed at 898c8a6 .